app/job/rest.py

```diff
@@ -132,6 +137,15 @@ def create_job(service_id):
     data["template"] = data.pop("template_id")
     template = dao_get_template_by_id(data["template"])
 
+    if template.template_type == SMS_TYPE:
+        job = get_job_from_s3(service_id, data["id"])
+        recipient_csv = RecipientCSV(
+            job,
+            template_type=template.template_type,
+            placeholders=template._as_utils_template().placeholders,
```

Comment:
Need to pass in template for the fragment count to be calculated properly here.

```suggestion
            placeholders=template._as_utils_template().placeholders,
            template=Template(template.__dict__),
```